charts/gardener-extension-admission-gcp/charts/application/templates/rbac.yaml

```diff
@@ -21,6 +21,34 @@ rules:
   - secrets
   verbs:
   - get
+  - list
+  - create
+  - patch
+  - watch
+- apiGroups:
+  - coordination.k8s.io
+  resources:
+  - leases
+  verbs:
+  - create
+- apiGroups:
+  - coordination.k8s.io
+  resources:
+  - leases
+  resourceNames:
+  - {{ include "name" . }}
+  verbs:
+  - get
+  - watch
+  - update
+- apiGroups:
+  - events.k8s.io
+  resources:
+  - events
+  verbs:
+  - create
+  - patch
+  - update
```

Comment:
Looks as if these changes have been applied by mistake in the provider-aws.
Please also remove them. See https://github.com/gardener/gardener-extension-provider-aws/pull/1100#pullrequestreview-2447758364
```suggestion
```